code/_onclick/hud/ghost.dm

```diff
@@ -59,3 +59,9 @@
 	using = new /obj/screen/ghost/teleport()
 	using.screen_loc = ui_ghost_teleport
 	static_inventory += using
+
+/datum/hud/ghost/show_hud()
```

Comment:
Hmmm, why is this needed when parent calls `update_parallax_pref`, which eventually uses `create_parallax`

Reply:
Removed this and it continued to work fine.